### PR TITLE
Implement ChatBanner component

### PR DESCRIPTION
**Pull request by Simhanischal**
Implement ChatBanner component

Screenshot:

image

### CODE REVIEW

File: src/components/chat/chatBanner/ChatBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <>
+      <div className="banner">
+        <div className="bannerAvatar">
+          <img alt="Chat Avatar" src='https://cdn4.iconfinder.com/data/icons/logos-3/600/React.js_logo-1024.png' width="20" />
+        </div>
+        <div className="bannerBotDetails">
+          <Typography variant="body2">Mini ChatGPT</Typography>
+          <Typography
+            variant="caption"
+            style={{ fontSize: 11, color: green[300] }}
```
Comment (medhapatil12) [inline src/components/chat/chatBanner/ChatBanner.tsx:17]: should we move this styling also to the scss file?

Comment (Simhanischal) [reply src/components/chat/chatBanner/ChatBanner.tsx:17]: These styles are associated with Material UI components and cannot be set using a style file.

File: src/components/chat/chatBanner/ChatBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import Typography from "@material-ui/core/Typography";
+import { green } from "@material-ui/core/colors";
+import '../../../assets/stylesheets/chat/chatBanner.scss'
+
+const ChatBanner = () => {
+  return (
+    <>
+      <div className="banner">
+        <div className="bannerAvatar">
+          <img alt="Chat Avatar" src='https://cdn4.iconfinder.com/data/icons/logos-3/600/React.js_logo-1024.png' width="20" />
```
Comment (medhapatil12) [inline src/components/chat/chatBanner/ChatBanner.tsx]: we can import existing logo from assets/images instead of hardcoding the URL.

Comment (Simhanischal) [reply src/components/chat/chatBanner/ChatBanner.tsx]: nice catch! addressed https://github.com/Simhanischal/Mini-ChatGPT/commit/c4184336518cc40c5e2c1869abff852d53611f2d

**Review by medhapatil12 — APPROVED** [review]

**Pull request merged**